source/elements/oneMKL/source/domains/dft/compute_backward.rst

```diff
@@ -108,13 +108,15 @@ compute_backward (Buffer version)
    inout_re
       ``sycl::buffer`` object of sufficient capacity to store the elements
       defining the real parts of all the relevant data sequences, as configured
-      by ``desc``. Only with complex descriptors configured for in-place operations with
+      by ``desc``. ``data_type`` must be single or double precision floating-point, as 
+      described by the descriptor's precision. Only with complex descriptors configured for in-place operations with
```

Comment:
Nitpicking, AFAIK the convention in the spec is to format the lines to 80 characters max although it's not respected everywhere. The Rewrap extension works like a charm to manually format blocks of code if you're using VS Code.

Reply:
The rewrap extension does work like a charm. I've applied it to the changes in [4e444ba](https://github.com/uxlfoundation/oneAPI-spec/pull/544/commits/4e444ba34dff2e1f9bd1bb1966acd4b3754b1ea6).